lib/widgets/unread_count_badge.dart

```diff
@@ -65,3 +66,22 @@ class UnreadCountBadge extends StatelessWidget {
           count.toString())));
   }
 }
+
+class MutedUnreadBadge extends StatelessWidget {
+  const MutedUnreadBadge({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Opacity(
+      opacity: 0.5,
+      child: Container(
+        width: 8,
+        height: 8,
+        margin: const EdgeInsetsDirectional.only(end: 3),
+        decoration: const BoxDecoration(
+          color: HslColor(0, 0, 80),
+          shape: BoxShape.circle)));
```

Comment:
How about putting the 0.5 opacity in the `BoxDecoration.color`, with `HSLColor.fromHSLA(…).toColor()`?